battery_wall.sh

```diff
@@ -2,33 +2,51 @@
 
 ## Author : Aditya Shakya (adi1090x)
 
-DIR="/usr/share/battery-wallpaper"
+case "$OSTYPE" in
+	darwin*) DIR="/usr/local/share/battery-wallpaper" ;;
+	linux*) DIR="/usr/local/battery-wallpaper" ;;
```

Comment:
it should be ***/usr/share/battery-wallpaper*** according to changes you made in install.sh